### ISSUE TITLE
Can't host sixpack-web in relative URL

### ISSUE THREAD

**Issue by aquach** [seatgeek/sixpack#221]
I'm trying to run sixpack-web on https://domain.com/sixpack/* but all of its paths and assets assume that it's been hosted at the root. It'd be useful if it either allowed total customization of the mount path, or just didn't assume it was at the root.

**Comment by josegonzalez** [comment]
Would you be interested in creating a patch for this?

**Comment by aquach** [comment]
I might once we get to know the system more. We're just trying it out for now, and if we end up depending on it a lot, we'll try to make it better :)

**Comment by shahar3012** [comment]
You can run a reverse proxy with ngnix or Apache and set the root path to proxy to the web port and the /api path proxy to the server port.

**Comment by aquach** [comment]
Right, we already do that. The issue is that the paths generated in the HTML assume that Sixpack is mounted on the root. For example: assume Sixpack is hosted at https://www.domain.com/sixpack via nginx. You visit https://www.domain.com/sixpack, and at the top, there's a link that takes you back to the root. Unfortunately, that link is https://www.domain.com/, because Sixpack assumes that it's been mounted at the root.

**Comment by shahar3012** [comment]
I think it an has an easy fix, to change the links to be relative. In the file layout.html in sixpack -> templates directories at the line with brand class the href should be "./", rather than "/". If you want you can make a Pull request with this change, if not I'll.

**Comment by zackkitzmiller** [comment]
@shahar3012 I'd gladly accept a patch that allows relative URLs. It'd be a welcome contribution for sure.

**Comment by aquach** [comment]
@shahar3012 just to clarify, the top link is just one of many places where an absolute URL is assumed. There are a bunch of images, assets, and other links that are also affected.

In addition, there are issues with subfolders that just using ./ won't fix. For example, both the URL https://www.domain.com/sixpack/experiments/my-experiment and https://www.domain.com/sixpack/ accesses https://www.domain.com/sixpack/static/img/favicon.png. If you change the layout to refer to one with ./ or ../, the other will break because the relative origin is different (experiments/ vs /).

I could be wrong, but I think the only workable solution is to allow configuration of the absolute root and prepend it to any links that currently use a absolute path.

**Comment by shahar3012** [comment]
@aquach I think you are right.
I didn't get into much of the code, just tried to find exactly what is the issue.

**Comment by willwest** [comment]
I've made a bunch of changes on a forked repo to accomplish a slightly-working version of this, but there are a lot of broken bits and pieces. I unfortunately don't know enough about web dev to get a clean patch in a PR, though. Any help would be welcomed!

**Comment by josegonzalez** [comment]
@willwest if you make a pr, we can work on cleaning it up as we go.
